dataset-metadata/metadata.ts

```diff
@@ -232,7 +232,7 @@ const metadataForDataset = (datasetId: DatasetId): DatasetMetadata => ({
     datasetId === 'gnomad_sv_r4',
   isV2: datasetId.startsWith('gnomad_r2'),
   isV3: datasetId.startsWith('gnomad_r3'),
-  isV4: datasetId.startsWith('gnomad_r4'),
+  isV4: datasetId.startsWith('gnomad_r4') || datasetId.includes('r4'),
```

Comment:
The left operand can be removed (`startsWith`...) now since it is made redundant by the right one.